examples/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+cmake_minimum_required(VERSION 3.16.3)
+project(PCTExamples)
+
+set(ExampleSpecificComponents
+  PCT
+  )
+
+if(NOT ITK_SOURCE_DIR)
+  find_package(ITK REQUIRED COMPONENTS ITKImageIO ITKTransformIO ${ExampleSpecificComponents})
```

Comment:
Why would we need all these ITK components? Let's wait for a concrete example...